assets/js/script.js

```diff
@@ -63,47 +68,58 @@ function unixConversion(unix) {
 // Start of trails function
 //When the user gives the state code he needs to be able to pull a list of trails in the given state 
 
+
 var searchBtn = document.querySelector("#searchBtn");
-searchBtn.addEventListener("click", function () {
+searchBtn.addEventListener("click", getTrails);
+var menuList = document.querySelector(".menu");
+
+function displayTrails(){
     var selectState = document.querySelector("#selectState");
     var state = selectState.options[selectState.selectedIndex].value  //because options is an array,selected index is the index of the one we have currently selelcted
-    var apiUrl = "https://developer.nps.gov/api/v1/places?statecode=" + state + "&limit=50&q=trails&api_key=WdgBOclP1YDr6ZIL0vXfInjZRVwmb8VjKrcvwpoZ"
-
+    var apiUrl = "https://developer.nps.gov/api/v1/places?statecode=" + state + "&limit=300&q=trails&api_key=WdgBOclP1YDr6ZIL0vXfInjZRVwmb8VjKrcvwpoZ"
```

Comment:
300 responses seems to pull in the trails from all across U.S?

Reply:
Because some states have up to 280+ trails I was making sure that if I load those they will all display not just 50.